src/app/base/components/bricks-integrator/helper/dummy-nodes.ts

```diff
@@ -215,4 +215,8 @@ def vader_sentiment(record):
         elif vs["compound"] <= -0.05:
             return "negative"
     elif MODE == "scores": 
-        return vs`;
\ No newline at end of file
+        return vs`;
+
+
+// TODO: Remove this when grdp is removed from bricks
+export const REMOVE_GROUP_NODES = ['gdpr_compliant', 'not_gdpr_compliant'];
```

Comment:
personal prefernce:
Since this is a const I'd go with GROUPS_TO_REMOVE or BLACKLISTED_GROUPS since REMOVE_GROUP_NODES sounds like a function. Yes, the capitalization should also "hint" at a const value and it's also temporary so no real problem.